django/dispatch/dispatcher.py

```diff
@@ -64,10 +66,6 @@ def connect(self, receiver, sender=None, weak=True, dispatch_uid=None):
 
                 Receivers must be able to accept keyword arguments.
 
-                If receivers have a dispatch_uid attribute, the receiver will
```

Comment:
My understanding is that this comment refers to this code: https://github.com/django/django/blob/f8e4e4a935512aedf815f6aee85bc797b498ea75/django/dispatch/dispatcher.py#L130-L131